README.md

```diff
@@ -8,17 +8,28 @@ Include this repository as a module in your existing terraform code:
 
 ```
 module "build" {
-  source    = "git::https://github.com/cloudposse/tf_codebuild.git"
-  namespace = "general"
-  name      = "ci"
-  stage     = "staging"
-
-  image         = "apline"
-  instance_size = "BUILD_GENERAL1_SMALL"
+    source              = "git::https://github.com/cloudposse/tf_codebuild.git?ref=tags/0.5.0"
+    namespace           = "general"
+    name                = "ci"
+    stage               = "staging"
+    
+    image               = "aws/codebuild/docker:1.12.1"
+    instance_size       = "BUILD_GENERAL1_SMALL"
+    
+    # These attributes are optional, used as ENV variables when building Docker images and pushing them to ECR
+    # For more info:
+    # http://docs.aws.amazon.com/codebuild/latest/userguide/sample-docker.html
+    # https://www.terraform.io/docs/providers/aws/r/codebuild_project.html
+    
+    privileged_mode     = true
+    aws_region          = "us-east-1"
+    aws_account_id      = "xxxxxxxxxx"
+    image_repo_name     = "ecr-repo-name"
+    image_tag           = "latest"
 }
 ```
 
-Grant appropriate permsissions to s3
+Grant the required permissions to s3
```

Comment:
Can't we just make the module do this rather than exposing it as documentation?

Reply:
Removed the `S3` permissions since they are not related to this module